### DISCUSSION TITLE
how to select with condition?

### DISCUSSION THREAD

**Discussion by imknight** [PostgREST/postgrest#1969] [Q&A] [answered]
Hi looking for some help on how to query the API

- how do I query with API for SQL "Select * from projects where user_id in ( select * from users )"
- how do I query with the relation key? I manage to have /rest/v1/projects?select=*,development(name) to get all projects with development, how can I match it with development name?

**Comment by steve-chavez** [comment, marked as answer]
Hey @imknight,

> how do I query with API for SQL "Select * from projects where user_id in ( select * from users )"

You'll need to create a view for this case, see [Custom Queries](https://postgrest.org/en/v8.0/api.html#custom-queries).

> how do I query with the relation key? I manage to have /rest/v1/projects?select=*,development(name) to get all projects with development, how can I match it with development name?

Check [Embedded filters](https://postgrest.org/en/v8.0/api.html#embedded-filters).

**Reply by imknight** [reply]
thanks! really appreciate it.
